languages/php/menhir/parsing_hacks_php.ml

```diff
@@ -128,10 +128,10 @@ let find_paren_tokens toks replace =
     | x :: xs -> (
         match x with
         | TCPAR t ->
-            let x' = if depth == 0 && replace then T_LAMBDA_CPAR t else x in
+            let x' = if depth =|= 0 && replace then T_LAMBDA_CPAR t else x in
             aux xs (x' :: acc) (depth + 1)
         | TOPAR t ->
-            if depth == 1 then
+            if depth =|= 1 then
               let x' = if replace then T_LAMBDA_OPAR t else x in
               (List.rev (x' :: acc), xs)
             else aux xs (x :: acc) (depth - 1)
```

Comment:
semgrep bug? These two occurrences of `==` should have been caught by https://semgrep.dev/playground/r/ocaml.lang.correctness.physical-vs-structural.physical-equal

Reply:
excluded from the scan by semgrep.jsonnet